templates/contribute/naming.md

```diff
@@ -70,76 +111,23 @@ When translating the statements of theorems into words, this dictionary is often
 | `/`    |                       | `div`         |                                                             |
 | `•`    | `\bu`                 | `smul`        |                                                             |
 | `⁻¹`   | `\-1`                 | `inv`         |                                                             |
-| `⅟`    | `\frac1`              | `inv_of`      |                                                             |
+| `⅟`    | `\frac1`              | `invOf`       |                                                             |
 | `∣`    | <code>\\&#124;</code> | `dvd`         |                                                             |
 | `∑`    | `\sum`                | `sum`         |                                                             |
 | `∏`    | `\prod`               | `prod`        |                                                             |
 
 #### Lattices
 
-| symbol | shortcut | name             | notes |
-|--------|----------|------------------|-------|
-| `<`    |          | `lt`             |       |
-| `≤`    | `\le`    | `le`             |       |
-| `⊔`    | `\sup`   | `sup`            |       |
-| `⊓`    | `\inf`   | `inf`            |       |
-| `⨆`    | `\supr`  | `supr` / `bsupr` |       |
-| `⨅`    | `\infi`  | `infi` / `binfi` |       |
-| `⊥`    | `\bot`   | `bot`            |       |
-| `⊤`    | `\top`   | `top`            |       |
-
-### General conventions
-
-Identifiers are generally lower case with underscores. For the most
-part, we rely on descriptive names. Often the name of theorem simply
-describes the conclusion:
-
-- `succ_ne_zero`
-- `mul_zero`
-- `mul_one`
-- `sub_add_eq_add_sub`
-- `le_iff_lt_or_eq`
-
-If only a prefix of the description is enough to convey the meaning,
-the name may be made even shorter:
-
-- `neg_neg`
-- `pred_succ`
-
-Sometimes, to disambiguate the name of theorem or better convey the
-intended reference, it is necessary to describe some of the
-hypotheses. The word "of" is used to separate these hypotheses:
-
-- `lt_of_succ_le`
-- `lt_of_not_ge`
-- `lt_of_le_of_ne`
-- `add_lt_add_of_lt_of_le`
-
-Sometimes abbreviations or alternative descriptions are easier to work
-with. For example, we use `pos`, `neg`, `nonpos`, `nonneg` rather than
-`zero_lt`, `lt_zero`, `le_zero`, and `zero_le`.
-
-- `mul_pos`
-- `mul_nonpos_of_nonneg_of_nonpos`
-- `add_lt_of_lt_of_nonpos`
-- `add_lt_of_nonpos_of_lt`
-
-Sometimes the word "left" or "right" is helpful to describe variants
-of a theorem.
-
-- `add_le_add_left`
-- `add_le_add_right`
-- `le_of_mul_le_mul_left`
-- `le_of_mul_le_mul_right`
-
-An injectivity lemma that uses "left" or "right" should refer to the
-argument that "changes". For example, a lemma with the statement
-`a - b = a - c ↔ b = c` could be called `sub_right_inj`.
-
-We can also use the word "self" to indicate a repeated argument:
-
-- `mul_inv_self`
-- `neg_add_self`
```

Comment:
Don't these largely still apply?

Reply:
I removed them because I think they are largely copied below already (let me know if you think otherwise)